### DISCUSSION TITLE
Schematic:  Transform node

### DISCUSSION THREAD

**Discussion by F-Burning** [opentoonz/opentoonz#6728] [Ideas]
A node to transform the image (position, scale, rotation...) Davinci Resolve or Natron (among many other node systems), have a transform node, and I think it can be useful in many aspects, such as creating chromatic aberration without moving the column, position/scale/rotate multiple images at the same time, etc. Besides, with this node keyframes are no longer created automatically.

**Comment by artisteacher** [comment]
I have definitely wished this was an option a few times.

**Comment by F-Burning** [comment]
> I have definitely wished this was an option a few times.

It would be very useful. The only problem is that it would not be in real time, but it can be worth it.

**Comment by RodneyBaker** [comment]
Interestingly, as of the last version -Toonz Premium- has a translate node.
It doesn't have the other transforms however.

**Comment by F-Burning** [comment]
Just now I was trying to move the Glare Iwa effect using the animation tool on the column, but it's not possible, so I think I'd could with a transform node.

![011](https://user-images.githubusercontent.com/74098734/208317271-0f32bac3-d4c6-4afa-a36f-46808453cc9b.jpg)

 This is only an example of the importance of this node.

**Comment by RodneyBaker** [comment]
Perhaps @shun-iwasawa can request the source for their Translate FX in Toonz Premium and then we can develop it into a more robust set of transform nodes.  That would seem to be a more direct way to proceed as well as a good excuse to keep Digital Video in the development loop.

**Comment by RodneyBaker** [comment]
I should add that many FX rely on columns so moving the columns with transform will automatically move the FX.  Glare FX does not appear to be one of those.
